lib/Account.php

```diff
@@ -201,6 +256,8 @@ public static function allCapabilities($id, $params = null, $opts = null)
         return self::_allNestedResources($id, static::PATH_CAPABILITIES, $params, $opts);
     }
 
+    const PATH_EXTERNAL_ACCOUNTS = '/external_accounts';
```

Comment:
This was useful when we were writing code manually. I wonder if it isn't cleaner/easier to just have the route in the function now.

Reply:
Stupid PHP question: would removing those constants be a backwards-breaking change? Not sure if they are public or not.